package.json

```diff
@@ -3,16 +3,20 @@
   "version": "1.1.2",
   "author": "Fabrique numérique des Ministères Sociaux <dsi-incubateur@sg.social.gouv.fr> (https://www.fabrique.social.gouv.fr)",
   "devDependencies": {
-    "@babel/cli": "7.12.13",
-    "@babel/preset-env": "7.12.13",
     "@socialgouv/eslint-config-react": "1.49.0",
-    "eslint": "7.19.0",
-    "esm": "3.2.25",
+    "@socialgouv/eslint-config-typescript": "^1.51.0",
+    "@types/jest": "26.0.8",
+    "@typescript-eslint/eslint-plugin": "3.8.0",
+    "@typescript-eslint/parser": "3.8.0",
+    "eslint": "7.1.0",
+    "eslint-plugin-import": "2.22.0",
```

Comment:
```suggestion
```

Already in [`@socialgouv/eslint-config-typescript`](https://github.com/SocialGouv/linters/blob/master/packages/eslint-config-typescript/package.json#L12)